### DISCUSSION TITLE
Leave device in AP mode

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16951] [Q&A] [answered]
I know I can use the wificonfig command to set AP mode which then re-connects it to the existing wifi settings.
Is there a way of putting the device into AP mode and it stays there until it is configured without going back to the old settings after a time out period.

**Comment by barbudor** [comment, marked as answer]
To prevent Tasmota from rebooting and connect to AP you should clear the SSID (both 1 and 2 if you have both set)
Something like
```
Backlog SSID1 1; SSID2 1; WifiConfig 2
```
Should do the job

**Reply by dac2020** [reply]
Would it be possible to do this from the broker / mqtt client?
I mean, what would the syntax be if it was possible.

cmnd/<TOPIC>/Backlog   SSID1 1; SSID2 1; WifiConfig 2

**Reply by barbudor** [reply]
You are missing the topic for the device you want to change
`cmnd/devicetopic/Backlog` and `SSID1 1;SSID2 1; WifiConfig 2`
Did you 1st tried in the console ?

**Reply by dac2020** [reply]
Works fine from console, just wondering if it could be done from remote, using broker/client only.
Basically, something like this. But, have not found a way to send multiple commands in one.

I do have topic but github removes it :).

```
cmnd/<TOPIC>/Backlog   SSID1 1; SSID2 1; WifiConfig 2
```

**Reply by barbudor** [reply]
Everything that you do in the console can be done through MQTT

**Reply by dac2020** [reply]
I understand that but I was not sure if this is one of those so wanted to ask.

**Reply by sfromis** [reply]
You *had* found the way of sending multiple commands in one, the usual `Backlog`. Just make sure to have the Backlog command name as part of the topic, and the semicolon-delimited command list as the payload.

**Reply by dac2020** [reply]
Ok, I'll try again as this did not work which is why I was asking.
Is xmnd/topic/backlog correct

**Reply by sfromis** [reply]
Yes, `cmnd/<TOPIC>/Backlog` is valid in the topic part the MQTT message, as the "Topic" used in Tasmota is just the variable part of the MQTT topics. And then you need the rest in the payload.

**Reply by dac2020** [reply]
Ah, semicolon typo, works now. Thank you.
